org.audiveris.audiveris.metainfo.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.audiveris.audiveris</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0</project_license>
+  <name>Audiveris</name>
+  <summary>Optical Music Recognition (OMR) Application</summary>
+  <description>
+    <p>
+      An Optical Music Recognition (OMR) application, featuring
+      an OMR engine coupled with a dedicated editor.
+    </p>
+    <p>
+      It recognizes music from digitized images and converts it into
+      computer-readable symbolic format. This enables further music processing
+      by any external notation editor, most notably: digital playback,
+      transposition and arrangement.
+    </p>
+    <p>
+      Audiveris provides outputs in two main digital formats: its OMR format
+      and standard MusicXML format. It has a graphical user
+      interface focused on quick verification and manual
+      correction of the OMR outputs. Other music editors, such as MuseScore,
+      Finale, or Lilypond, can be used on Audiveris MusicXML output.
+    </p>
+  </description>
+  <categories>
+    <category>Music</category>
+    <category>Scanning</category>
+    <category>AudioVideo</category>
+  </categories>
+  <url type="homepage">https://audiveris.github.io/audiveris/</url>
+  <url type="bugtracker">https://github.com/Audiveris/audiveris/issues</url>
+  <url type="help">https://audiveris.github.io/audiveris/_pages/handbook/</url>
+  <url type="contact">https://github.com/Audiveris/audiveris/discussions</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://commons.wikimedia.org/wiki/File:Audiveris_5.2.png</image>
+      <caption>Audiveris 5.2 editor with scanned sheet music</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://commons.wikimedia.org/wiki/File:Audiveris_5.2_screenshot.png</image>
```

Comment:
Screenshots should come from upstream

Reply:
The upstream [documentation](https://audiveris.github.io/audiveris/) currently doesn't have good screen shots for this purpose, AFAICS, and I am not sure if the ones that exist have table URLs.

Wikimedia commons is currently the best we can get.